nabcommon/nabservice.py

```diff
@@ -147,6 +147,20 @@ async def stop_service_loop(self):
         """
         return None
 
+    @staticmethod
```

Comment:
I don't know Python that much, so I'm not sure it's the best way to do this. The idea is to have a central place holding the hostname of the nabd daemon.